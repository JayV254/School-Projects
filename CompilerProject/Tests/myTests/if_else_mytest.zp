int main() {
        int a;
        int b;
	int result;

        a = 3;
        b = 10;
        result = a * b;

        if (result < 20) {
                puts("a times b is less than 20");
        }
        else {
                puts("a times b is greater than 20");
        }

        return 0;
}

