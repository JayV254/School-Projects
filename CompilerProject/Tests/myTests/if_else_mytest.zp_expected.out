a times b is greater than 20
